test/models/backend_api_config_test.rb

```diff
@@ -5,18 +5,18 @@ def setup
     @config = BackendApiConfig.new
```

Comment:
Do not forget to do and test also this: https://github.com/3scale/porta/pull/1308#issuecomment-540135411

Reply:
btw also this might not be necessary anymore to convert:
https://github.com/3scale/porta/blob/734c29a867ae443da0c54c17a04d310a338b51b6/app/models/backend_api_config.rb#L17